packages/run-protocol/test/amm/vpool-xyk-amm/test-xyk-amm-swap.js

```diff
@@ -1168,8 +1168,9 @@ test('amm adding liquidity', async t => {
   const timer = buildManualTimer(console.log, 30n);
 
   const { zoe, amm } = await setupAmmServices(electorateTerms, centralR, timer);
+  // verify that addPool works with ERef<Issuer>.
```

Comment:
not worth another commit but this comment seems to be about 1173 not 1172

Reply:
There was an `expect-error` for this issue that I neglected to remove, so I had another chance to make a change.